--- steam-audio/core/deps/flatbuffers/include/flatbuffers/flatbuffers.h
+++ steam-audio/core/deps/flatbuffers/include/flatbuffers/flatbuffers.h
@@ -1873,6 +1873,3 @@
    private:
-    TableKeyComparator &operator=(const TableKeyComparator &other) {
-      buf_ = other.buf_;
-      return *this;
-    }
+    FLATBUFFERS_DELETE_FUNC(TableKeyComparator &operator=(const TableKeyComparator &other))
   };
